### PR TITLE
fix transformation of facets

### PR DESCRIPTION
**Pull request by luca-penasa**
- change min-max values in cell fusion dlg
- normal are shown as sqrt(polyline-length)

### CODE REVIEW

File: plugins/qFacets/cellsFusionDlg.ui [hunk 1]
```diff
@@ -299,9 +305,7 @@ Output facet normals may be randomly oriented
    <header location="global">ccOctreeSpinBox.h</header>
   </customwidget>
  </customwidgets>
- <resources>
-  <include location="qFacets.qrc"/>
```
Comment (luca-penasa) [inline plugins/qFacets/cellsFusionDlg.ui]: For some reason qtdesigner removed this. It was not intended but it did that by its own.

Comment (cloudcompare) [inline plugins/qFacets/cellsFusionDlg.ui]: Can you fix this?

File: libs/qCC_db/ccFacet.cpp [hunk 2]
```diff
@@ -415,7 +415,11 @@ void ccFacet::drawMeOnly(CC_DRAW_CONTEXT& context)
 		markerContext.display = 0;
 
 		c_unitNormalSymbol->setTempColor(m_contourPolyline->getColor());
-		PointCoordinateType scale = m_contourPolyline->getOwnBB().getMinBoxDim();
+
+        //		PointCoordinateType scale = m_contourPolyline->getOwnBB().getMinBoxDim();
+
+        // maybe using surface sqrt is better?
+        PointCoordinateType scale = sqrt(m_contourPolyline->computeLength());
```
Comment (asmaloney) [inline libs/qCC_db/ccFacet.cpp]: It looks like your space/tabs are off throughout this PR.

**Comment by luca-penasa** [comment]
I'll put all together in a single commit :-)

**Pull request closed without merging**